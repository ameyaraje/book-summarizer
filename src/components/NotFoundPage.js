import React from 'react';
import 'bootstrap/dist/css/bootstrap.css';

const NotFoundPage = () => {
    return (
        <div className="container">
            <h2>
                This route does not exist! Please check the URL.
            </h2>
        </div>
    );
};

export default NotFoundPage;