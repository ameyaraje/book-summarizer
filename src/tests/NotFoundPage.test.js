import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import NotFoundPage from '../components/NotFoundPage';

test('should render Not Found Page correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NotFoundPage />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});

