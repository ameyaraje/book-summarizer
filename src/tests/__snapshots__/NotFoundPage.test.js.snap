// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Not Found Page correctly 1`] = `
<div
  className="container"
>
  <h2>
    This route does not exist! Please check the URL.
  </h2>
</div>
`;
